src/tests/Microsoft.PowerFx.Core.Tests/IntellisenseTests/SuggestTest.cs

```diff
@@ -34,13 +35,35 @@ private string[] SuggestStrings(string expression, PowerFxConfig config, string
             return intellisense.Suggestions.Select(suggestion => suggestion.DisplayText.Text).ToArray();
         }
 
-        private readonly PowerFxConfig _default = PowerFxConfig.BuildWithEnumStore(null, new EnumStoreBuilder().WithDefaultEnums());
```

Comment:
>private readonly PowerFxConfig _default = [](http://example.com/codeflow?start=8&length=41)

If goal here is to just change from a field to a property, can you just change = to => ?

        private readonly PowerFxConfig _default => PowerFxConfig.BuildWithEnumStore(null, new EnumStoreBuilder().WithDefaultEnums());